lte/gateway/c/core/oai/tasks/nas/esm/sap/esm_sap.hpp

```diff
@@ -64,5 +63,3 @@ status_code_e esm_sap_send(esm_sap_t* msg);
 #ifdef __cplusplus
```

Comment:
Same as above

Reply:
This will be done in a separate PR that I am working on